tests/api/circ_policies/test_circ_policies_rest.py

```diff
@@ -278,14 +278,15 @@ def test_circ_policy_secure_api_update(client,
     # special case : cipo at library_level
     record_url = url_for('invenio_records_rest.cipo_item',
                          pid_value=circ_policy_temp_martigny.pid)
-    login_user_via_session(client, librarian_martigny.user)
+    login_user_via_session(client, system_librarian_martigny.user)
```

Comment:
According to new roles-rights guide, there are no more special case for "library level" cipo, this special test can be removed. Additionally, four **`*_secure_api`** methods from this file can be removed too and **`test_circ_policies_permissions`** (first method of this file) too.